temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging. Use a logging framework instead to provide better control over log levels and log outputs.
```python
import logging
logging.basicConfig(level=logging.INFO)
logging.info('hello')
```